lib/pbench/test/unit/agent/task/test_results_move.py

```diff
@@ -121,6 +124,49 @@ def test_move_results(monkeypatch, caplog, pytestconfig):
         )
 
         runner = CliRunner(mix_stderr=False)
+
+        # Test --no-delete
+        result = runner.invoke(
+            move.main,
+            args=[
+                TestMoveResults.CTRL_SWITCH,
+                TestMoveResults.CTRL_TEXT,
+                TestMoveResults.TOKN_SWITCH,
+                TestMoveResults.TOKN_TEXT,
+                TestMoveResults.DELN_SWITCH,
+            ],
+        )
+        assert (
+            result.exit_code == 0
+        ), f"Expected a successful operation, exit_code == {result.exit_code:d}"
+        assert (
+            result.stdout
+            == "Status: total # of result directories considered 1, successfully copied 1, encountered 0 failures\n"
+        )
+        # This should raise an unexpected exception if it was not created.
+        (pbrun / f"{name}.copied").unlink()
+
+        # Test --delete (default) with .running directory.
+        (pbrun / name / ".running").mkdir()
+        result = runner.invoke(
+            move.main,
+            args=[
+                TestMoveResults.CTRL_SWITCH,
+                TestMoveResults.CTRL_TEXT,
+                TestMoveResults.TOKN_SWITCH,
+                TestMoveResults.TOKN_TEXT,
+            ],
+        )
+        assert (
+            result.exit_code == 1
+        ), f"Expected an unsuccessful operation, exit_code == {result.exit_code:d}"
+        assert (
+            result.stdout
+            == "Status: total # of result directories considered 1, successfully moved 0, encountered 1 failures\n"
+        )
+        (pbrun / name / ".running").rmdir()
+
+        # Test --delete (default).
```

Comment:
Do you want to test the `--delete`-omitted case?

Reply:
Why wouldn't we?

Reply:
> Why wouldn't we?

I took this response as "yes, we should add that", but I don't see the corresponding change to the code.

Reply:
Neither `DELY_SWITCH` nor `DELN_SWITCH` appears in the options here which may have been Peter's intent; however with the `.running` directory, I'm not sure this is quite the test Webb envisioned as nothing happens to confirm proper defaulting. The following test case, however, seems to fit, confirming `successfully moved 1` vs [...] `copied 1`.

Reply:
OK, I think maybe I am/was confused.

It now looks like `test_args()` tests the `--delete` case; and `test_results_move()` tests the `--no-delete` case, along with the default case with and without a `.running` directory present.  So, I think it's covered.